packages/alchemy/src/middleware/gasManager.ts

```diff
@@ -40,14 +55,32 @@ export interface AlchemyGasEstimationOptions {
 const dummyPaymasterAndData =
   <C extends ClientWithAlchemyMethods>(client: C) =>
   () => {
-    switch (client.chain.id) {
-      case 1:
-      case 10:
-      case 137:
-      case 42161:
-        return "0x4Fd9098af9ddcB41DA48A1d78F91F1398965addcfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff0000000000000000000000000000000007aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa1c";
+    const paymasterAddress = "0x4Fd9098af9ddcB41DA48A1d78F91F1398965addc";
+    const olderPaymasterAddress = "0xc03aac639bb21233e0139381970328db8bceeb67";
+    const arbSepoliaPaymasterAddress =
+      "0x0804Afe6EEFb73ce7F93CD0d5e7079a5a8068592";
+
+    const dummyData =
```

Comment:
same - global constant? what accounts does this work and not work for?

Reply:
let's keep this in the dummy constant for now since it's not used elsewhere